test/unit/stubs.dart

```diff
@@ -0,0 +1,112 @@
+// Copyright 2016 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:async';
+
+import 'package:test/test.dart';
+import 'package:w_common/disposable.dart';
+
+import 'typedefs.dart';
+
+class DisposableThing extends Object with Disposable {
+  bool wasOnDisposeCalled = false;
+
+  void testManageDisposable(Disposable thing) {
```

Comment:
If you're feeling ambitious, these `test*` methods exist because the `manage*` methods used to be `@protected`, which is no longer the case. So you could remove them and update the tests.